client/src/assets/Logo.svg

```diff
@@ -1,32 +1,32 @@
 <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1858.527 320.167">
   <defs>
     <style>
-      .cls-1, .cls-3 {
+      .log-1, .log-2, .log-3, .log-4 {
         fill: #fff;
       }
 
-      .cls-2, .cls-3 {
+      .log-2, .log-3 {
         isolation: isolate;
       }
 
-      .cls-3 {
+      .log-3 {
```

Comment:
Why do we have multiple CSS classes with the same values?

Reply:
This is an auto generated svg, I changed the names to stop conflicts but no way am I optimising it by hand lmao